syllabus/5-Substrate/3-Merklized-Storage_Slides.md

```diff
@@ -217,11 +235,11 @@ flowchart TD
 
 Notes:
 
-more resources:
+More resources:
 
-- https://en.wikipedia.org/wiki/Merkle_tree
-- https://en.wikipedia.org/wiki/Radix_tree
-- https://en.wikipedia.org/wiki/Trie
+- <https://en.wikipedia.org/wiki/Merkle_tree>
+- <https://en.wikipedia.org/wiki/Radix_tree>
+- <https://en.wikipedia.org/wiki/Trie>
```

Comment:
I'm not sure why you altered the links here. These are inside the notes and would show correctly. Adding the <> makes them actually wrong
```suggestion
- https://en.wikipedia.org/wiki/Merkle_tree
- https://en.wikipedia.org/wiki/Radix_tree
- https://en.wikipedia.org/wiki/Trie
```

Reply:
I am not really sure what happened there :/. Fixed it now.